pss/pss_test.go

```diff
@@ -1836,11 +1802,30 @@ func newServices(allowRaw bool) adapters.Services {
 		params.MaxRetries = 1000
 		params.RetryExponent = 2
 		params.RetryInterval = 1000000
-		kademlias[id] = network.NewKademlia(id[:], params)
+		kademlias[id] = network.NewKademlia(bzzKey, params)
 		return kademlias[id]
 	}
-	return adapters.Services{
-		protocolName: func(ctx *adapters.ServiceContext) (node.Service, error) {
+	return map[string]simulation.ServiceFunc{
+		"bzz": func(ctx *adapters.ServiceContext, bucket *sync.Map) (s node.Service, cleanup func(), err error) {
+			addr := network.NewAddr(ctx.Config.Node())
+			bzzPrivateKey, err := simulation.BzzPrivateKeyFromConfig(ctx.Config)
+			if err != nil {
+				return nil, nil, err
+			}
+			addr.OAddr = network.PrivateKeyToBzzKey(bzzPrivateKey)
```

Comment:
This is the overlay address object we need to set in the kademlia. Now the kademlia constructor above is using a different one. this will be solved when kademlia is concealed by the bzz object when we separate it to a separate "service", but I would like it correct in the meantime. Perhaps you can use the bucket?

Reply:
Cleared up in call, we don't need to touch this, never mind